knowledge_graph/classifier/embedding.py

```diff
@@ -20,26 +25,67 @@ class EmbeddingClassifier(Classifier, ZeroShotClassifier):
     def __init__(
         self,
         concept: Concept,
-        embedding_model_name: str = "BAAI/bge-small-en-v1.5",
+        embedding_model_name: str = "ibm-granite/granite-embedding-107m-multilingual",
         threshold: float = 0.65,
+        document_prefix: str = "",
+        query_prefix: str = "",
```

Comment:
not sure i understand what these two new kwargs are doing - why are they necessary? what do they add?